source/overview/compliance.rst

```diff
@@ -139,7 +137,7 @@ Mattermost Enterprise Edition software is found under `ECCN 5D002 <https://www.b
 
 The other key areas to be aware of for an export of Mattermost software or technology are:
 
-**Sanctions**: There are comprehensive sanctions to Cuba, Iran, North Korea, Sudan, Syria and other countries/territories
+**Sanctions**: There are comprehensive sanctions to Cuba, Iran, North Korea, Syria, and other countries/territories
```

Comment:
@jamiehurewitz - Confirming that Sudan is omitted from this list intentionally.

Reply:
Yes, Sudan has been removed from the embargo list since they overthrew their government and created a democratic government.